src/__tests__/ceramic_integration.test.ts

```diff
@@ -127,7 +131,10 @@ async function makeCAS(
   configCopy.blockchain.connectors.ethereum.network = 'ganache'
   configCopy.blockchain.connectors.ethereum.rpc.port = minConfig.ganachePort + ''
   const childContainer = container.createChildContainer()
```

Comment:
I think it'd be cleaner if you made `childContainer` an argument to the makeCAS function, instead of a return value.  You can make and store a handle to the container in the test setup function, then pass it into makeCAS